core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>org.fusesource.ide.core</groupId>
+    <artifactId>plugins</artifactId>
+    <version>11.1.0-SNAPSHOT</version>
+    <relativePath>..</relativePath>
+  </parent>
+
+  <artifactId>org.fusesource.ide.camel.model.service.impl.v2210fuse710018redhat00001</artifactId>
+  <packaging>eclipse-plugin</packaging>
+  <name>Red Hat Fuse Tooling :: Core :: Plugins :: Camel Dynamic Service Implementation for 2.21.0.fuse-710018-redhat-00001</name>
+  
+  <properties>
+  		<bundle.camel.version>2.21.0.fuse-710018-redhat-00001</bundle.camel.version>
+  		<bundle.slf4j.version>1.7.22</bundle.slf4j.version>
+  </properties>
+  <build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>get-deps</id>
+						<goals>
+							<goal>copy</goal>
+						</goals>
+						<phase>generate-resources</phase>
+					</execution>
+				</executions>
+				<configuration>
+					<artifactItems>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-catalog</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+							<artifactId>camel-catalog-maven</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+							<artifactId>camel-catalog-provider-karaf</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+							<artifactId>camel-catalog-provider-springboot</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-core</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-spring</artifactId>
+							<version>${bundle.camel.version}</version>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.apache.camel</groupId>
+            				<artifactId>camel-blueprint</artifactId>
```

Comment:
indention

Reply:
fixed, don't know why Github still showing the comment